src/pages/WalletsStart/WalletsStart.js

```diff
@@ -0,0 +1,69 @@
+// @flow strict
+
+import React, {
+  Fragment,
+  Component,
+} from 'react'
+
+import { t } from 'ttag'
+
+import { WalletsCreate } from 'pages/WalletsCreate/WalletsCreate'
+import { WalletsImport } from 'pages/WalletsImport/WalletsImport'
+
+import {
+  LogoHeader,
+  NewWalletButtons,
+} from 'components'
+
+import walletsStartStyle from './walletsStart.m.scss'
+
+export type WalletAction = 'CREATE' | 'IMPORT'
+type WalletActions = { [WalletAction]: WalletAction }
+
+type Props = {||}
```

Comment:
I'm not sure, but I think that `null` is better, because `null` comparing is simpler.